### PR TITLE
Bug 1277029: edit link to storage options

### PR DESCRIPTION
**Pull request by bfallonf**
@miheer PR is in regard to BZ 1277029:

https://bugzilla.redhat.com/show_bug.cgi?id=1277029

Can I get an ack this is all that's needed for the BZ? The info from the training docs you've linked in the BZ is in the storage section (from what I can tell), and as there are more than just NFS available now, I thought it best to link to the index rather than the specific NFS topic.

### CODE REVIEW

**Comment by bfallonf** [comment]
@miheer Updated with an example for PVCs as well. Please, let me know if this satisfies the BZ.

**Comment by bfallonf** [comment]
@miheer has given the thumbs up in the BZ. 

@tpoitras @tnguyen-rh @adellape available for peer review :cool:

File: install_config/install/docker_registry.adoc [hunk 1]
```diff
@@ -95,11 +95,18 @@ Any images anyone has built or pushed into the registry would disappear.
...
+link:../../install_config/persistent_storage/index.html[use the persistent
+storage method of your choice].
+
+For example, to use an existing persistent volume claim:
+----
+$ oc volume deploymentconfigs/docker-registry --add --name=v1 -t pvc \
+     --claim-name=<pvc_name> --overwrite
+----
+
+Or, to attach an existing NFS volume to the registry once it has been
```
Comment (tnguyen-rh) [inline install_config/install/docker_registry.adoc]: What does "it" refer to, the "existing NFS volume" or the "registry"?

**Comment by tnguyen-rh** [comment]
@bfallonf Just a small nit (from a genuinely ignorant pov).  Otherwise, LGTM.

File: install_config/install/docker_registry.adoc (hunk 1 above)
Comment (adellape) [inline install_config/install/docker_registry.adoc]: Think the original intent here was referring to the registry. Kinda strange to mention that here but not in the preceding PVC example. Maybe move the " to the registry once it has been defined" sentiment up before both examples, in the "For production use" intro sentence?

**Comment by adellape** [comment]
@bfallonf This has a conflict now because I fixed a typo in https://github.com/openshift/openshift-docs/pull/1285 in that sessionaffinity workaround section, which we were both touching. When you rebase/resolve conflicts, please make sure we don't revert back to the incorrect "ClientAPI" (as discussed in https://github.com/openshift/openshift-docs/pull/1285#discussion-diff-46440944). :bow:

**Comment by bfallonf** [comment]
@tnguyen-rh @adellape Yeah I moved it around a little. Better?

**Comment by tnguyen-rh** [comment]
@bfallonf LGTM.

**Comment by bfallonf** [comment]
Cool merging. Thanks!

**Pull request merged**